ai_serving/fasttext_clf/fasttext_clf.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: fasttext-clf
+spec:
+  selector:
+    pod: fasttext-clf
+  ports:
+    - protocol: TCP
+      port: 4675
+      targetPort: 4675
+      name: http
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fasttext-clf-up
+  namespace: default
+data:
+  prod_up: |-
+    mkdir /dl; cd /dl; git clone https://prod:$GITPASS@git.jaseci.org/marsninja/jaseci.git;
+    mv jaseci/ai_serving/ /;
+    cd /ai_serving;
+    python3 -m fasttext_clf.fasttext_clf_app;
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: jsuse-git-pass
+type: Opaque
+data:
+  password: UHIwZE4xbmo0
+---
+apiVersion: v1
```

Comment:
Also don't need the docker registry credentials either. When this is ready to be merged, I will build the docker image and push to the jaseci public dockerhub https://hub.docker.com/repository/docker/jaseci/ai-service